### ISSUE TITLE
AndroidTV app reports wrong playback time for a livestream that is transcoding or remuxing

### ISSUE THREAD

**Issue by timminator** [jellyfin/jellyfin-androidtv#4303]
### Describe the bug

1. Watch a livestream that is transcoding or remuxing inside the Jellyfin AndroidTV app.
2. Check the stream in the Jellyfin Dashboard via a Browser, playback time reported for the stream looks something like this: "481,744:09:39/0:00"

Instead i expect something like "12:34/0:00".

On the old version i used before (0.17.9) this did also not work correctly. It was not correctly differentiating between transcoding and remuxing and playback time just jumped between 0 and 2 seconds. In the new version the detection of remuxing works correctly but the playback time is wrong.

This works correctly when watching the stream in  a browser or other clients.

### Application version

0.18.2

### Where did you install the app from?

Amazon Appstore

### Device information

Amazon Fire TV Stick 4K Gen. 2

### Android version

Android Level 30 (Android 11)

### Jellyfin server version

10.10.3

**Comment by nielsvanvelzen** [comment]
The app reports the correct position. A livestream is not _really_ a livestream but (simply said) a video that keeps getting extended with new frames. This means that often when you "start" watching you're actually just seeking to the current end of the video.

The app is supposed to report the current playback position to the server and it's then up to the server/other clients on how to display the information. The web client knows if it is displaying information for a livestream and could chose to not show the progress but apparently it does not.

**Comment by timminator** [comment]
Sorry for reviving this thread, but something is definitely not correct. I made an image of what I'm seeing:

![Image](https://github.com/user-attachments/assets/d8654066-5cb3-41f7-a653-65532f36cfc6)

When watching the same stream in the browser it shows the the time the person watched the stream for, in this case 33 seconds. As you explained there is no total time as it's just a video that keeps getting extended, so "0:33/0:00" makes sense to me.

But the reported watch time for the Jellyfin AndroidTV app is totally off and does not reflect the time the person watched the livestream for - in this case around one minute.

Now tested on version 0.18.4 I also noticed that when clicking on the info button for the AndroidTV device it reports that the livestream is transcoding but it is just remuxing as confirmed by the server logs and also by previously disabling transcoding for the user.

I hope this explains my issue better.
